src/OrchardCore.Modules/Orchard.Autoroute/Drivers/AutoroutePartDisplay.cs

```diff
@@ -61,14 +65,16 @@ public override IDisplayResult Edit(AutoroutePart autoroutePart)
         public override async Task<IDisplayResult> UpdateAsync(AutoroutePart model, IUpdateModel updater)
         {
             await updater.TryUpdateModelAsync(model, Prefix, t => t.Path);
-            
+
             var httpContext = _httpContextAccessor.HttpContext;
-            
             if (httpContext != null && await _authorizationService.AuthorizeAsync(httpContext.User, Permissions.SetHomepage))
             {
                 await updater.TryUpdateModelAsync(model, Prefix, t => t.SetHomepage);
             }
 
+            var validationErrorKeyPrefix = string.IsNullOrEmpty(Prefix) ? string.Empty : Prefix + ".";
```

Comment:
I wasn't able to find any example for the `ContentPartDriver`. Content field drivers use 
`updater.ModelState.AddModelError(Prefix, T["Message"])` but now I see that, it probably doesn't work correctly ... It shows validation errors in the summary section, but not next to the field.

![image](https://cloud.githubusercontent.com/assets/2894161/24927815/8c26e73a-1f00-11e7-8a73-ceeaf63bc759.png)

with `validationErrorKeyPrefix` it show the error in the summary as well as next to the field

![image](https://cloud.githubusercontent.com/assets/2894161/24928007/2f2b2c02-1f01-11e7-8dfe-52eb985e58e4.png)

So we might need some helper that creates the correct prefix ...